Week2/homework/maartjes-work.js

```diff
@@ -1,65 +1,80 @@
-'use strict';
-
-const monday = [
-  {
-    name: 'Write a summary HTML/CSS',
-    duration: 180,
-  },
-  {
-    name: 'Some web development',
-    duration: 120,
-  },
-  {
-    name: 'Fix homework for class10',
-    duration: 20,
-  },
-  {
-    name: 'Talk to a lot of people',
-    duration: 200,
-  },
-];
-
-const tuesday = [
-  {
-    name: 'Keep writing summary',
-    duration: 240,
-  },
-  {
-    name: 'Some more web development',
-    duration: 180,
-  },
-  {
-    name: 'Staring out the window',
-    duration: 10,
-  },
-  {
-    name: 'Talk to a lot of people',
-    duration: 200,
-  },
-  {
-    name: 'Look at application assignments new students',
-    duration: 40,
-  },
-];
-
-const maartjesTasks = monday.concat(tuesday);
-const maartjesHourlyRate = 20;
-
-function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
-}
-
-// eslint-disable-next-line no-unused-vars
-const earnings = computeEarnings(maartjesTasks, maartjesHourlyRate);
-
-// add code to convert `earnings` to a string rounded to two decimals (euro cents)
-
-console.log(`Maartje has earned €${'replace this string with the earnings rounded to euro cents'}`);
-
-// Do not change or remove anything below this line
-module.exports = {
-  maartjesTasks,
-  maartjesHourlyRate,
-  computeEarnings,
-};
+'use strict';
+
+const monday = [
+  {
+    name: 'Write a summary HTML/CSS',
+    duration: 180,
+  },
+  {
+    name: 'Some web development',
+    duration: 120,
+  },
+  {
+    name: 'Fix homework for class10',
+    duration: 20,
+  },
+  {
+    name: 'Talk to a lot of people',
+    duration: 200,
+  },
+];
+
+const tuesday = [
+  {
+    name: 'Keep writing summary',
+    duration: 240,
+  },
+  {
+    name: 'Some more web development',
+    duration: 180,
+  },
+  {
+    name: 'Staring out the window',
+    duration: 10,
+  },
+  {
+    name: 'Talk to a lot of people',
+    duration: 200,
+  },
+  {
+    name: 'Look at application assignments new students',
+    duration: 40,
+  },
+];
+
+const maartjesTasks = monday.concat(tuesday);
+const maartjesHourlyRate = 20;
+
+function computeEarnings(tasks, hourlyRate) {
+  let result = tasks.map(toHours => toHours.duration / 60);
+  console.log(result); // [3, 2, 0.33, 3.35, 4, 3, 0.16, 3.35, 0.66]
+  result = result.filter(twoHoursOrMore => twoHoursOrMore >= 2);

```

Comment:
Same goes for this parameter. An array of durations that are `twoHoursOrMore` is the result you get from this filter.
I would call the item that get's passed in this filter `durationInHours`.

Reply:
done